guidelines/sc/22/consistent-help.html

```diff
@@ -15,7 +15,7 @@ <h4>Consistent Help</h4>
     </ul>
 
   <p class="note">Access to help mechanisms may be provided directly on the page, or may be provided via a direct link to a different page containing the information.</p>
-  <p class="note">For this Success Criterion, the same relative order can be thought of as how the content is ordered when the page is serialized. The visual position of a help mechanism is likely to be consistent across pages for the same page variation (e.g., CSS break-point). The user can cause a change, for example zoom or change orientation, which may trigger a different page variation.</p>
+  <p class="note">For this Success Criterion, the same relative order can be thought of as how the content is ordered when the page is serialized. The visual position of a help mechanism is likely to be consistent across pages for the same page variation (e.g., CSS break-point). The user can cause a change in which page variation is displayed, for example by changing a page's zoom level or switching the orientation of a page, which would be regarded as user-initiated changes.</p>
```

Comment:
`a change in which page variation is displayed` sounds a bit awkward and confusing to me, and the whole sentence is perhaps a bit "back to front".

how about turning it around to something more direct:

`If a different page variation is displayed when the user changes the page's zoom level, or changes the orientation of their device. this is regarded as a user-initiated change.`